libloki/api.js

```diff
@@ -216,57 +172,15 @@
     });
     return syncMessage;
   }
-  async function sendPairingAuthorisation(authorisation, recipientPubKey) {
-    const pairingAuthorisation = createPairingAuthorisationProtoMessage(
-      authorisation
-    );
-    const ourNumber = textsecure.storage.user.getNumber();
-    const ourConversation = await ConversationController.getOrCreateAndWait(
-      ourNumber,
-      'private'
-    );
-    // Send
-    const p = new Promise((resolve, reject) => {
-      const callback = result => {
-        // callback
-        if (result.errors.length > 0) {
-          reject(result.errors[0]);
-        } else {
-          resolve();
-        }
-      };
-      const pairingRequestMessage = textsecure.OutgoingMessage.buildPairingRequestMessage(
-        recipientPubKey,
-        ourNumber,
-        ourConversation,
-        authorisation,
-        pairingAuthorisation,
-        callback
-      );
-
-      pairingRequestMessage.sendToNumber(recipientPubKey, false);
-    });
-    return p;
-  }
-
-  function sendSessionRequestsToMembers(members = []) {
-    // For every member, see if we need to establish a session:
-    members.forEach(memberPubKey => {
-      const haveSession = _.some(
-        textsecure.storage.protocol.sessions,
-        s => s.number === memberPubKey
-      );
-
+  async function sendSessionRequestsToMembers(members = []) {
+    // For every member, trigger a session request if needed
+    members.forEach(async memberStr => {
       const ourPubKey = textsecure.storage.user.getNumber();
-      if (!haveSession && memberPubKey !== ourPubKey) {
-        // eslint-disable-next-line more/no-then
-        ConversationController.getOrCreateAndWait(memberPubKey, 'private').then(
-          () => {
-            const sessionRequestMessage = textsecure.OutgoingMessage.buildSessionRequestMessage(
-              memberPubKey
-            );
-            sessionRequestMessage.sendToNumber(memberPubKey);
-          }
+      if (memberStr !== ourPubKey) {
+        await ConversationController.getOrCreateAndWait(memberStr, 'private');
```

Comment:
We should be fine not creating a conversation

Reply:
fixed